.github/workflows/ci-weekly.yml

```diff
@@ -4,6 +4,9 @@ on:
   push: 
     branches:
     - master
+  pull_request:
+    branches:
+    - master
```

Comment:
Once this is approved a final commit should be made to remove `pull_request`.  It's just being used for testing, @Ibrahimmaga for the reminder.